gen/definitions/virtual_network_to_fabric_site.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Virtual Network To Fabric Site
```

Comment:
Nice name. 👍

Reply:
in API its not intuitive. You have: 
- Add virtual network with scalable groups (/dna/intent/api/v1/virtual-network)
- Add VN in fabric (/dna/intent/api/v1/business/sda/virtual-network)

Second resource is used to add VN (created using first resource) to fabric site.